test/suites/profiling.bash

```diff
@@ -20,8 +20,12 @@ SUITE_profiling_PROBE() {
 }
 
 SUITE_profiling_SETUP() {
-    echo 'int main(void) { return 0; }' >test.c
     unset CCACHE_NODIRECT
+
+    cat <<EOF >test.c
+int main(void) { return 0; }
+EOF
+    backdate test.c
```

Comment:
As mentioned in https://github.com/ccache/ccache/pull/1033#issuecomment-1114030692, this hides a real problem instead of fixing it.

Reply:
I'll rebase your branch to remove abad5b0c46b6e31e53d1fef3dcbb7d105bb45a6b since I want to apply f4f6e470ed0e396017b3eba90c9a9fa21d0529cf.